transaction-aggregation-decisioning-processor/src/clients/arangodb.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
please remove ArangoDB as this service doesn't use it. Unnecessary DB connection.